### ISSUE TITLE
Show task tooltips on keyboard navigation

### ISSUE THREAD

**Issue by chuckn0rris** [bryntum/support#4722]
[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=56&t=21421&p=106045#p106045)

We have customised tooltips that work on mouse hover, however it does not work with keyboard navigation. I have attached a video showing this within the example taskboard - https://bryntum.com/examples/taskboard/tooltips/
